cypress/e2e/transaction-import.cy.ts

```diff
@@ -163,11 +161,43 @@ describe('Transaction Import', () => {
     cy.awaitLoading()
 
     cy.getInputFor('Account').type('My account{enter}')
-    cy.getInputFor('File').selectFile(
-      'cypress/fixtures/transactions-parsed.csv'
-    )
+    cy.getInputFor('File').selectFile('cypress/fixtures/comdirect.csv')
 
     cy.clickAndWait('Submit')
     cy.contains('This is a duplicate of an existing transaction')
   })
+
+  it('errors on already parsed file', function () {
+    cy.get('nav').contains('Transactions').click()
+    cy.getByTitle('Import Transactions').click()
+    cy.awaitLoading()
+
+    cy.getInputFor('Account').type('account')
+    cy.contains('External Account').should('not.exist')
+    cy.contains('My Other Account')
+    cy.contains('My Account').click()
+
+    cy.getInputFor('File').selectFile('cypress/fixtures/comdirect-ynap.csv')
+
+    cy.contains('Submit').click()
+    cy.contains('Error: This file has already been converted')
+  })
+
+  it('errors on an unparseable file', function () {
+    cy.get('nav').contains('Transactions').click()
+    cy.getByTitle('Import Transactions').click()
+    cy.awaitLoading()
+
+    cy.getInputFor('Account').type('account')
+    cy.contains('External Account').should('not.exist')
+    cy.contains('My Other Account')
+    cy.contains('My Account').click()
+
+    cy.getInputFor('File').selectFile(
+      'cypress/fixtures/transactions-no-parser.csv'
+    )
+
+    cy.contains('Submit').click()
+    cy.contains('Error: No parser is available for this file')
```

Comment:
Same here. It should say something like "we can't parse this file, please check here (link to ynap) if your bank is supported"